playground/legacy/__tests__/legacy.spec.ts

```diff
@@ -48,8 +53,7 @@ test('generates assets', async () => {
           'immutable-chunk: 404',
           'immutable-chunk-legacy: 404',
           'polyfills-legacy: 404'
-        ].join('\n'),
-    true
+        ].join('\n')
```

Comment:
Is this change intended?

Reply:
yes, because the test already write the sever and build mode expected value.

Reply:
Then, could we remove the `isBuild` condition passed to `untilUpdated` args?

Reply:
maybe not. the test will run two times. build for one expect value. serve for other expect value

Reply:
If you are intending to run this test case to run in both of them, don't we need to leave the third argument(`true`)?

Reply:
yes.

Reply:
I think I'm not understanding your intention here. You mean `true` should be removed and `isBuild` should exist?

If `true` does not exist, this `untilUpdate` will do nothing during build test. So I think `isBuild` can be removed.

Reply:
oh sorry. I make a mistask🤦